lib/stylable.rb

```diff
@@ -17,9 +17,11 @@ def stylable
                    writeable: true,
                    readable: true,
                    getter: lambda { |args| stylings.collect{|s| s.style} },
-                   setter: lambda { |val, args|
-                     styling = stylings.find_or_initialize_by(style: val)
-                     stylings << styling unless styling.persisted?
```

Comment:
This line wasn't having any effect -- the deleted line 21 was already adding the new styling to the `stylings` association.